### PR TITLE
add metadata object identifiers to uri translation method

### PR DESCRIPTION
**Pull request by sadam21**

### CODE REVIEW

File: src/main/java/com/gooddata/md/MetadataService.java [hunk 1]
```diff
@@ -341,4 +336,36 @@ public void removeObjByUri(String uri) {
 
         return uris;
     }
+
+    /**
+     * Find metadata URIs by restrictions. Identifier is the only supported restriction.
```
Comment (martiner) [inline src/main/java/com/gooddata/md/MetadataService.java]: fix "restrictions"

File: src/test/java/com/gooddata/ShowcaseAT.java [hunk 2]
```diff
@@ -354,6 +354,11 @@ public void loadDatasetBatch() throws Exception {
         datasetService.loadDatasets(project, personManifest, cityManifest).get();
     }
 
+    @Test(groups = "dataset", dependsOnMethods = "createModel")
+    public void identifiersToUri() {
+        assertThat(gd.getMetadataService().identifiersToUris(project, asList("attr.person.department")).size(), is(1));
```
Comment (martiner) [inline src/test/java/com/gooddata/ShowcaseAT.java]: add better assert. eg get the returned uri and compare the identifier

File: src/main/java/com/gooddata/md/MetadataService.java (hunk 1 above)
Comment (sadam21) [inline src/main/java/com/gooddata/md/MetadataService.java]: Javadoc fixed.

File: src/test/java/com/gooddata/ShowcaseAT.java (hunk 2 above)
Comment (sadam21) [inline src/test/java/com/gooddata/ShowcaseAT.java]: Added...

File: src/test/java/com/gooddata/ShowcaseAT.java [hunk 3]
```diff
@@ -354,6 +365,16 @@ public void loadDatasetBatch() throws Exception {
         datasetService.loadDatasets(project, personManifest, cityManifest).get();
     }
 
+    @Test(groups = "dataset", dependsOnMethods = "createModel")
```
Comment (martiner) [inline src/test/java/com/gooddata/ShowcaseAT.java]: group should be md

**Pull request merged**